network-api/networkapi/buyersguide/pagemodels/product_update.py

```diff
@@ -1,7 +1,11 @@
 from django.db import models
 
+from wagtail.snippets.models import register_snippet
+from wagtail.search import index
 
-class Update(models.Model):
+
+@register_snippet
+class Update(index.Indexed, models.Model):
```

Comment:
TIL: index

Reply:
Will this also make products show up when you do regular CMS search? (worried about getting 3 pages and 300 product results when staff searches =D)

Reply:
Yep! This makes it "searchable" in the modal when using a SnippetChooserPanel 🎉